examples/template/__tests__/test-template.ava.js

```diff
@@ -24,8 +24,8 @@ test.beforeEach(async t => {
     // Deploy and init the counter JS contract
     const counter = await root.createSubAccount('counter');
     let contract_base64 = (await readFile('build/contract.base64')).toString();
-    await counter.call(jsvm, 'deploy_js_contract', Buffer.from(contract_base64, 'base64'), {attachedDeposit: '400000000000000000000000'});
-    await counter.call(jsvm, 'call_js_contract', encodeCall(counter.accountId, 'init', []), {attachedDeposit: '400000000000000000000000'});
+    await counter.call(jsvm, 'deploy_js_contract', Buffer.from(contract_base64, 'base64'), {attachedDeposit: '800000000000000000000000'});
```

Comment:
The contract is 37! times bigger now :)
I think it's better not to support Borsh at all. But I believe we will be able to find the solution.

Reply:
Yes this is how commonjs module worse than esm, rollup cannot handle just importing the used functions. And you found it's even worse that it also packaging all borsh code :(